### ISSUE TITLE
PHP 7 and apcu \OC\Memcache\APCu not available for local cache (Not a duplicate)

### ISSUE THREAD

**Issue by tabp0le** [owncloud/core#21994]
I'm aware of the current posted issue with PHP 7 and apcu, #21095, which leads me to this one..

I have PHP 7.0.2 and apcu installed. I installed the patch for incompatibility. OwnCloud loads fine, everything seems to be working, but..

**Steps to reproduce**
1. Setup apcu php extension
2. Make sure it's configured with apc.enable_cli=1
3. Setup apcu for locking and local memcache in owncloud config.
4. Apply patch; #21308 to use correct APCu API
5. Set error reporting to include info.
6. Run cron.
7. Look at log.

**Expected behaviour**
No errors in log

**Actual behaviour**
Errors in log. 
`OC\HintException: Memcache \OC\Memcache\APCu not available for local cache at /var/www/vhosts/cloud.domain.net/httpdocs/lib/private/memcache/factory.php#94`
`Info	cli	Memcache \OC\Memcache\APCu not available for distributed cache	2016-01-28T17:37:48+00:00
Info	cli	Memcache \OC\Memcache\APCu not available for local cache`

**Server configuration**

**Operating system:** CloudLinux 6.7

**Web server:** nginx 1.9.9 & Apache/2.2.15 w/php-fpm enabled

**Database:** mysql  Ver 14.14 Distrib 5.6.26

**PHP version:** 7.0.2

**APCu Version:** 5.1.3

**ownCloud version:** 8.2.2 (stable)

**Updated from an older ownCloud or fresh install:**Update

**List of activated apps:**
```
$ ./occ app:list
Enabled:
  - activity: 2.1.3
  - bookmarks: 0.6
  - calendar: 0.8.1
  - contacts: 0.5.0.0
  - documents: 0.11.1
  - external: 1.2
  - files: 1.2.0
  - files_external: 0.3.0
  - files_pdfviewer: 0.7
  - files_sharing: 0.7.0
  - files_texteditor: 2.0
  - files_trashbin: 0.7.0
  - files_versions: 1.1.0
  - files_videoviewer: 0.1.3
  - firstrunwizard: 1.1
  - flowupload: 0.1
  - gallery: 14.3.0
  - music: 0.3.8
  - notifications: 0.1.0
  - ownnote: 1.05
  - piwik: 0.1.0
  - provisioning_api: 0.3.0
  - search_lucene: 0.6.0
  - templateeditor: 0.1
  - updater: 0.6
  - user_external: 0.4
Disabled:
  - encryption
  - files_videojs-sublime
  - galleryplus
  - html5_videoplayer
  - libreonline
  - mail
  - tasks
  - transmission
  - user_ldap
```

**The content of config/config.php:**
```
$ ./occ config:list system
{
    "system": {
        "instanceid": "ocfaaab88190",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.mydomain.net"
        ],
        "datadirectory": "\/var\/www\/vhosts\/cloud.mydomain.net\/ocdata",
        "overwrite.cli.url": "https:\/\/cloud.mydomain.net",
        "dbtype": "mysql",
        "version": "8.2.2.2",
        "dbname": "mydbt",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "forcessl": true,
        "asset-pipeline.enabled": false,
        "mail_from_address": "no-reply",
        "mail_smtpmode": "php",
        "mail_domain": "cloud.mydomain.net",
        "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
        "preview_max_scale_factor": 1,
        "enabledPreviewProviders": {
            "0": "OC\\Preview\\PNG",
            "1": "OC\\Preview\\JPEG",
            "2": "OC\\Preview\\GIF",
            "11": "OC\\Preview\\Illustrator",
            "12": "OC\\Preview\\Postscript",
            "13": "OC\\Preview\\Photoshop",
            "14": "OC\\Preview\\TIFF",
            "15": "OC\\Preview\\MSOffice2003",
            "16": "OC\\Preview\\MSOffice2007",
            "17": "OC\\Preview\\MSOfficeDoc",
            "18": "OC\\Preview\\OpenDocument",
            "19": "OC\\Preview\\PDF",
            "20": "OC\\Preview\\Photoshop",
            "21": "OC\\Preview\\Postscript",
            "22": "OC\\Preview\\StarOffice",
            "23": "OC\\Preview\\SVG",
            "24": "OC\\Preview\\RAW",
            "25": "OC\\Preview\\DNG",
            "26": "OC\\Preview\\MP4"
        },
        "tempdirectory": "\/tmp",
        "updatechecker": true,
        "has_internet_connection": true,
        "appstoreenabled": true,
        "appstoreurl": "https:\/\/api.owncloud.com\/v1",
        "appstore.experimental.enabled": true,
        "apps_paths": [
            {
                "path": "\/var\/www\/vhosts\/cloud.mydomain.net\/httpdocs\/apps",
                "url": "\/apps",
                "writable": true
            }
        ],
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\APCu",
        "app.mail.server-side-cache.enabled": true,
        "xframe_restriction": false,
        "enable_avatars": true,
        "theme": "spryCloud",
        "maintenance": false,
        "secret": "***REMOVED SENSITIVE VALUE***",
        "loglevel": 1,
        "trashbin_retention_obligation": "auto"
    }
}
```

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no
**
Are you using an external user-backend, if yes which one:** No
**
Client configuration**

**Browser:** Chrome 47.0.2526.111

**Operating system:** Windows 10

**Logs**
**Web server error log**
No errors
**`ownCloud log (data/owncloud.log)`**
```
OC_Template::printErrorPage('Memcache \\\\\\\\OC\\\\\\\\Me...', 'Is the matching...')\\n#18 {main}\",\"File\":\"\\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/memcache\\\/factory.php\",\"Line\":94}","level":3,"time":"2016-01-28T17:37:04+00:00"}
{"reqId":"v2QK3myVlPH2dYuCVVA9","remoteAddr":"63.229.64.134","app":"PHP","message":"Undefined index: defaultValue at \/var\/www\/vhosts\/cloud.mydomain.net\/httpdocs\/core\/ajax\/appconfig.php#47","level":3,"time":"2016-01-28T17:37:05+00:00"}
{"reqId":"VqpRwn8AAAEAB2LZPLoAAAAH","remoteAddr":"2607:ff68:109::5","app":"PHP","message":"OC\\HintException: Memcache \\OC\\Memcache\\APCu not available for local cache at \/var\/www\/vhosts\/cloud.mydomain.net\/httpdocs\/lib\/private\/memcache\/factory.php#94","level":3,"time":"2016-01-28T17:37:06+00:00"}
{"reqId":"I1ixfuWek4pcK8iuBP0y","remoteAddr":"","app":"cli","message":"Memcache \\OC\\Memcache\\APCu not available for local cache","level":1,"time":"2016-01-28T17:37:48+00:00"}
{"reqId":"I1ixfuWek4pcK8iuBP0y","remoteAddr":"","app":"cli","message":"Memcache \\OC\\Memcache\\APCu not available for distributed cache","level":1,"time":"2016-01-28T17:37:48+00:00"}
{"reqId":"069pU4DDbEIbJm1sjbzi","remoteAddr":"63.229.64.134","app":"PHP","message":"Array to string conversion at \/var\/www\/vhosts\/cloud.mydomain.net\/httpdocs\/lib\/private\/template\/functions.php#36","level":3,"time":"2016-01-28T17:37:57+00:00"}
{"reqId":"VqpR9X8AAAEAB2LSOTEAAAAA","remoteAddr":"63.229.64.134","app":"core","message":"Memcache \\OC\\Memcache\\APCu not available for local cache Is the matching PHP module installed and enabled?","level":3,"time":"2016-01-28T17:37:57+00:00"}
{"reqId":"VqpR9X8AAAEAB2LSOTEAAAAA","remoteAddr":"63.229.64.134","app":"core","message":"Exception: {\"Exception\":\"OC\\\\HintException\",\"Message\":\"Memcache \\\\OC\\\\Memcache\\\\APCu not available for local cache\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/server.php(268): OC\\\\Memcache\\\\Factory->__construct('4c36f789dd97472...', Object(OC\\\\Log), '\\\\\\\\OC\\\\\\\\Memcache\\\\\\\\AP...', NULL, '\\\\\\\\OC\\\\\\\\Memcache\\\\\\\\AP...')\\n#1 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/3rdparty\\\/pimple\\\/pimple\\\/src\\\/Pimple\\\/Container.php(112): OC\\\\Server->OC\\\\{closure}(Object(OC\\\\Server))\\n#2 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/appframework\\\/utility\\\/simplecontainer.php(104): Pimple\\\\Container->offsetGet('MemCacheFactory')\\n#3 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/server.php(754): OC\\\\AppFramework\\\\Utility\\\\SimpleContainer->query('MemCacheFactory')\\n#4 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/server.php(242): OC\\\\Server->getMemCacheFactory()\\n#5 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/3rdparty\\\/pimple\\\/pimple\\\/src\\\/Pimple\\\/Container.php(112): OC\\\\Server->OC\\\\{closure}(Object(OC\\\\Server))\\n#6 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/appframework\\\/utility\\\/simplecontainer.php(104): Pimple\\\\Container->offsetGet('URLGenerator')\\n#7 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/server.php(727): OC\\\\AppFramework\\\\Utility\\\\SimpleContainer->query('URLGenerator')\\n#8 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/helper.php(174): OC\\\\Server->getURLGenerator()\\n#9 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/template\\\/functions.php(169): OC_Helper::imagePath('', 'favicon.png')\\n#10 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/core\\\/templates\\\/layout.guest.php(15): image_path('', 'favicon.png')\\n#11 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/template\\\/base.php(160): include('\\\/var\\\/www\\\/vhosts...')\\n#12 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/template\\\/base.php(138): OC\\\\Template\\\\Base->load('\\\/var\\\/www\\\/vhosts...')\\n#13 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/template.php(211): OC\\\\Template\\\\Base->fetchPage()\\n#14 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/template.php(233): OC_Template->fetchPage()\\n#15 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/template\\\/base.php(121): OC_Template->fetchPage()\\n#16 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/template.php(308): OC\\\\Template\\\\Base->printPage()\\n#17 \\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/index.php(49): OC_Template::printErrorPage('Memcache \\\\\\\\OC\\\\\\\\Me...', 'Is the matching...')\\n#18 {main}\",\"File\":\"\\\/var\\\/www\\\/vhosts\\\/cloud.mydomain.net\\\/httpdocs\\\/lib\\\/private\\\/memcache\\\/factory.php\",\"Line\":94}","level":3,"time":"2016-01-28T17:37:57+00:00"}
{"reqId":"S\/KyuoueCCrMiQrgYKAr","remoteAddr":"63.229.64.134","app":"PHP","message":"Undefined index: defaultValue at \/var\/www\/vhosts\/cloud.mydomain.net\/httpdocs\/core\/ajax\/appconfig.php#47","level":3,"time":"2016-01-28T17:37:58+00:00"}
{"reqId":"VqpR938AAAEAB2LSOTQAAAAA","remoteAddr":"2607:ff68:109::5","app":"PHP","message":"OC\\HintException: Memcache \\OC\\Memcache\\APCu not available for local cache at \/var\/www\/vhosts\/cloud.mydomain.net\/httpdocs\/lib\/private\/memcache\/factory.php#94","level":3,"time":"2016-01-28T17:37:59+00:00"}
{"reqId":"\/A8ambBJAqbUrTTDIiyd","remoteAddr":"","app":"cli","message":"Memcache \\OC\\Memcache\\APCu not available for local cache","level":1,"time":"2016-01-28T17:45:01+00:00"}
{"reqId":"\/A8ambBJAqbUrTTDIiyd","remoteAddr":"","app":"cli","message":"Memcache \\OC\\Memcache\\APCu not available for distributed cache","level":1,"time":"2016-01-28T17:45:01+00:00"}
{"reqId":"NNOsifTa8U3kxx6BEPu0","remoteAddr":"","app":"cli","message":"Memcache \\OC\\Memcache\\APCu not available for local cache","level":1,"time":"2016-01-28T18:00:01+00:00"}
{"reqId":"NNOsifTa8U3kxx6BEPu0","remoteAddr":"","app":"cli","message":"Memcache \\OC\\Memcache\\APCu not available for distributed cache","level":1,"time":"2016-01-28T18:00:01+00:00"}
```
**Browser log**
```
`admin:47 GET https://cloud.mydomain.net/index.php/core/js/oc.js?v=453062c9fc47810810ed24fb740493ce 
shareconfigmodel.js?v=453062c9fc47810810ed24fb740493ce:22 Uncaught ReferenceError: oc_appconfig is not defined(anonymous function) @ shareconfigmodel.js?v=453062c9fc47810810ed24fb740493ce:22(anonymous function) @ shareconfigmodel.js?v=453062c9fc47810810ed24fb740493ce:83
jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6 XHR finished loading: GET "https://cloud.mydomain.net/core/img/breadcrumb.svg".send @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.extend.ajax @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6SVGSupport.checkMimeType @ js.js?v=453062c9fc47810810ed24fb740493ce:1168initCore @ js.js?v=453062c9fc47810810ed24fb740493ce:1283c @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4p.fireWith @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4x.extend.ready @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4q @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4
jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6 XHR finished loading: GET "https://cloud.mydomain.net/index.php/core/ajax/appconfig.php?app=piwik&key=piwik&action=getValue".send @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.extend.ajax @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.(anonymous function) @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.extend.getJSON @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6OC.AppConfig.getCall @ config.js?v=453062c9fc47810810ed24fb740493ce:14OC.AppConfig.getValue @ config.js?v=453062c9fc47810810ed24fb740493ce:37(anonymous function) @ settings-admin.js?v=453062c9fc47810810ed24fb740493ce:17c @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4p.fireWith @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4x.extend.ready @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4q @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4
jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6 XHR finished loading: GET "https://cloud.mydomain.net/index.php/avatar/tfitts/128".send @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.extend.ajax @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.(anonymous function) @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6$.fn.avatar @ jquery.avatar.js?v=453062c9fc47810810ed24fb740493ce:82(anonymous function) @ avatar.js?v=453062c9fc47810810ed24fb740493ce:4c @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4p.fireWith @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4x.extend.ready @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4q @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4
jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6 XHR finished loading: GET "https://cloud.mydomain.net/index.php/core/ajax/appconfig.php?app=piwik&key=internal&defaultValue=false&action=getValue".send @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.extend.ajax @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.(anonymous function) @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.extend.getJSON @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6OC.AppConfig.getCall @ config.js?v=453062c9fc47810810ed24fb740493ce:14OC.AppConfig.getValue @ config.js?v=453062c9fc47810810ed24fb740493ce:37(anonymous function) @ settings-admin.js?v=453062c9fc47810810ed24fb740493ce:40c @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4p.fireWith @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4x.extend.ready @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4q @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4
jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6 XHR finished loading: GET "https://cloud.mydomain.net/index.php/apps/notifications".send @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.extend.ajax @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6OCA.Notifications.fetch @ app.js?v=453062c9fc47810810ed24fb740493ce:283OCA.Notifications.initialFetch @ app.js?v=453062c9fc47810810ed24fb740493ce:140OCA.Notifications.initialise @ app.js?v=453062c9fc47810810ed24fb740493ce:59(anonymous function) @ app.js?v=453062c9fc47810810ed24fb740493ce:338c @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4p.fireWith @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4x.extend.ready @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4q @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4
angular.js?v=453062c9fc47810810ed24fb740493ce:9253 XHR finished loading: GET "https://cloud.mydomain.net/index.php/apps/updater/backup/index".(anonymous function) @ angular.js?v=453062c9fc47810810ed24fb740493ce:9253sendReq @ angular.js?v=453062c9fc47810810ed24fb740493ce:9087$http @ angular.js?v=453062c9fc47810810ed24fb740493ce:8878$http.(anonymous function) @ angular.js?v=453062c9fc47810810ed24fb740493ce:9021(anonymous function) @ updater.js?v=453062c9fc47810810ed24fb740493ce:140invoke @ angular.js?v=453062c9fc47810810ed24fb740493ce:2864instantiate @ angular.js?v=453062c9fc47810810ed24fb740493ce:2874(anonymous function) @ angular.js?v=453062c9fc47810810ed24fb740493ce:4759(anonymous function) @ angular.js?v=453062c9fc47810810ed24fb740493ce:4338forEach @ angular.js?v=453062c9fc47810810ed24fb740493ce:138nodeLinkFn @ angular.js?v=453062c9fc47810810ed24fb740493ce:4323compositeLinkFn @ angular.js?v=453062c9fc47810810ed24fb740493ce:3969nodeLinkFn @ angular.js?v=453062c9fc47810810ed24fb740493ce:4354compositeLinkFn @ angular.js?v=453062c9fc47810810ed24fb740493ce:3969nodeLinkFn @ angular.js?v=453062c9fc47810810ed24fb740493ce:4354compositeLinkFn @ angular.js?v=453062c9fc47810810ed24fb740493ce:3969publicLinkFn @ angular.js?v=453062c9fc47810810ed24fb740493ce:3874(anonymous function) @ angular.js?v=453062c9fc47810810ed24fb740493ce:963Scope.$eval @ angular.js?v=453062c9fc47810810ed24fb740493ce:8011Scope.$apply @ angular.js?v=453062c9fc47810810ed24fb740493ce:8091(anonymous function) @ angular.js?v=453062c9fc47810810ed24fb740493ce:961invoke @ angular.js?v=453062c9fc47810810ed24fb740493ce:2857resumeBootstrapInternal @ angular.js?v=453062c9fc47810810ed24fb740493ce:959bootstrap @ angular.js?v=453062c9fc47810810ed24fb740493ce:973angularInit @ angular.js?v=453062c9fc47810810ed24fb740493ce:934(anonymous function) @ angular.js?v=453062c9fc47810810ed24fb740493ce:14756c @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4p.fireWith @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4x.extend.ready @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4q @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4
jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6 XHR finished loading: PROPFIND "https://cloud.mydomain.net/remote.php/webdav".send @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.extend.ajax @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6OC.SetupChecks.checkWebDAV @ setupchecks.js?v=453062c9fc47810810ed24fb740493ce:37(anonymous function) @ admin.js?v=453062c9fc47810810ed24fb740493ce:170c @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4p.fireWith @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4x.extend.ready @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4q @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4
jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6 XHR finished loading: GET "https://cloud.mydomain.net/index.php/heartbeat".send @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6x.extend.ajax @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6OC.SetupChecks.checkGeneric @ setupchecks.js?v=453062c9fc47810810ed24fb740493ce:140(anonymous function) @ admin.js?v=453062c9fc47810810ed24fb740493ce:172c @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4p.fireWith @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4x.extend.ready @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4q @ jquery.min.js?v=453062c9fc47810810ed24fb740493ce:4
jquery.min.js?v=453062c9fc47810810ed24fb740493ce:6 XHR f`

**Comment by LukasReschke** [comment]
> I'm aware of the current posted issue with PHP 7 and apcu, which leads me to this one..

Please be more precise. Which issue? We have a ton of here :wink: 

Did you install https://github.com/krakjoe/apcu-bc?

**Comment by LukasReschke** [comment]
Also fill out the issue template. You don't say which version you use etc.

https://github.com/owncloud/core/blob/master/issue_template.md

**Comment by tabp0le** [comment]
Sorry about that. I'll fill it out. No I'm not using the apcu-bc, I thought with the patch released that was not required.

**Comment by LukasReschke** [comment]
> Sorry about that. I'll fill it out. No I'm not using the apcu-bc, I thought with the patch released that was not required.

What patch? :wink:

**Comment by tabp0le** [comment]
#21308

**Comment by tabp0le** [comment]
OK, I updated my report using the template. Sorry about that.

**Comment by casper000** [comment]
Memcache \\\\OC\\\\Memcache\\\\APCu not available for local cache\

have you tried to use redis not APCu?

**Comment by enoch85** [comment]
Just to confirm, Redis with PHP 7 works, and is the prefered cache instead of APCu. https://www.techandme.se/install-redis-cache-on-ubuntu-server-with-php-7-and-owncloud/

**Comment by LukasReschke** [comment]
Fixed in the 9.0 branches by https://github.com/owncloud/core/pull/21308. For 8.2 please install the bc package.
